app/models/announcement.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Announcement < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+
+  validates :author,
```

Comment:
Do you think we should limit the `message` length to 160 characters?

Reply:
We talked about that. Might not be a bad idea. The text field looks pretty big on the screen - they might be tempted to fill it up.

Reply:
+1 for limiting

Reply:
Will add

Reply:
put a lower limit of 5 chars on it. lemme know if you want it changed or dont want that at all :)